src/main/resources/site/pages/default/default.js

```diff
@@ -91,7 +91,8 @@ exports.get = function handleGet(request) {
     stk.data.deleteEmptyProperties(siteConfig);
     const content = portal.getContent();
 
-    const menuItems = menuLib.getMenuTree(3);
+    const menuItems = menuLib.getMenuTree(1);
```

Comment:
You can use lib-menu 4.0.0 just change to `getMenuTree(1, {})` its a known bug

Reply:
Will do. Next update.